package.json

```diff
@@ -6,7 +6,7 @@
   "dependencies": {
     "@ledgerhq/hw-transport-webusb": "^6.27.13",
     "@react-spring/web": "^9.6.1",
-    "@secretkeylabs/xverse-core": "1.6.1",
+    "@secretkeylabs/xverse-core": "1.6.1-a184dd9",
```

Comment:
Out of interest, which core PR is this?

Reply:
This was not relevant to the PR. The core version has been set back to 1.6.1. I was under the impression that the btc recovery feature was added in the beta version of core since it was added in the app side PR but talked with haseeb and it was added since it was for an app specific fix. Reverted the change in the PR

Reply:
Ah, awesome :)